pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -360,10 +362,16 @@ var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageCl
 		}
 	},
 	"driver.longhorn.io": func(sc *storagev1.StorageClass) string {
-		migratable := sc.Parameters["migratable"]
-		if migratable == "true" {
+		if sc.Parameters["migratable"] == "true" {
```

Comment:
does the order matter? couldn't it be for instance migratable and fs, in which case we'll return rwx

Reply:
As noted in the description, `migratable` is only supported for block, and I guess we would like to prefer it even if the SC is mis-configured with fsType.